interfaces/Portal/src/app/components/update-property-item/update-property-item.component.ts

```diff
@@ -66,11 +76,7 @@ export class UpdatePropertyItemComponent implements OnInit {
   ) {}
 
   ngOnInit() {
-    if (this.type === RegistrationAttributeType.MultiSelect) {
```

Comment:
what is the status of multiselect in 121. We're still not maintaining it, but cant it just be removed here?